helm/doc-ver/values.yaml

```diff
@@ -435,25 +444,163 @@ bundle-visual-anomaly-core-versions:
         maxReplicas: 3
       model:
         globalStorage:
-            type: gs
-            secret: sa-json
-            bucket: identity-enc-models-prod
+          type: gs
+          secret: sa-json
+          bucket: identity-enc-models-prod
         storage:
-            type: gs
-            secret: sa-json
-            bucket: identity-enc-models-prod
+          type: gs
+          secret: sa-json
+          bucket: identity-enc-models-prod
   models:
     6478fcb410dcce6d3b037199:
       # Model resources can be reduced, however, it is not recommended if
-      # the model is used in production. 
+      # the model is used in production.
       minLimits:
         cpu: 2
         memory: "2Gi"
       maxLimits:
-        cpu: 2
```

Comment:
I understand what you are doing. However, I don't think this will do what you expect as maxLimits have default value set in this chart. We can look into this another time

Reply:
I would modify the underlying charts as well to remove predefined cpu limits